osa_cli_releases/releasing.py

```diff
@@ -251,6 +251,8 @@ def update_ansible_role_requirements_file(
 
```

Comment:
There is a typo in the commit message, and I am not sure to understand how this whole story triggers, care to explain?

Reply:
So in a-r-r for the repo we define extra key `shallow_since`: https://review.opendev.org/c/openstack/openstack-ansible/+/804976/4/ansible-role-requirements.yml

If we see that key, we need to update the date with the latest commit date not to pull extra history.